Index: Readme.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Ambient Intelligence PPG Mini-Praktika\n\nMini-Praktikum: Venen Funktionsmessung mittels PPG\n\nZiel dieses Praktikums ist es herauszufinden ob man mit dem populären MAX30102 Sensor eine Venen PPG Messung durchführen kann.\n\n# Pull repository including submodules\n\n```\n git submodule update --init --recursive\n```\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Readme.md b/Readme.md
--- a/Readme.md	(revision fea4391f8fad3f33437fc8379298803340decabb)
+++ b/Readme.md	(date 1642069754184)
@@ -4,8 +4,9 @@
 
 Ziel dieses Praktikums ist es herauszufinden ob man mit dem populären MAX30102 Sensor eine Venen PPG Messung durchführen kann.
 
-# Pull repository including submodules
-
+## Pull repository including 
+Das Projekt beinhaltet einen seriellen RealTimePlotter, welcher über ein rekursives Repositody importiert werden muss. 
 ```
  git submodule update --init --recursive
 ```
+## Implementierung der PPG Messung
