Index: .gitmodules
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitmodules b/.gitmodules
new file mode 100644
--- /dev/null	(date 1642077065190)
+++ b/.gitmodules	(date 1642077065190)
@@ -0,0 +1,3 @@
+[submodule "heartrate_analysis_python"]
+	path = heartrate_analysis_python
+	url = https://github.com/paulvangentcom/heartrate_analysis_python.git
Index: heartrate_analysis.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/heartrate_analysis.py b/heartrate_analysis.py
new file mode 100644
--- /dev/null	(date 1642079942848)
+++ b/heartrate_analysis.py	(date 1642079942848)
@@ -0,0 +1,24 @@
+# Datei liest Daten aus einem CSV File und berechnet aus den Werten
+# Example heartrate anaylsis python
+
+import heartpy as hp
+import matplotlib.pyplot as plt
+
+
+def load_data():
+    path = 'readings/good_readings/data0.csv'
+    __data = hp.get_data(path)
+    return __data
+
+
+data, timer = hp.load_exampledata()
+
+# run the analysis
+wd, m = hp.process(data, sample_rate=100)
+
+# plt.figure(figsize=(12, 4))
+
+hp.plotter(wd,m )
+
+#plt.plot(data)
+#plt.show()
Index: readings/good_readings/README.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/readings/good_readings/README.md b/readings/good_readings/README.md
new file mode 100644
--- /dev/null	(date 1642157211666)
+++ b/readings/good_readings/README.md	(date 1642157211666)
@@ -0,0 +1,7 @@
+data0.csv enthält 2 mehr oder weniger gute PPG Messungen am linken Bein
+
+data1.csv
+
+data2.csv mit philipp am 14.01.
+
+data3.csv messung von philipp
\ No newline at end of file
Index: Readme.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Ambient Intelligence PPG Mini-Praktika\n\nPython Code zum Auslesen des Mikrocontrollers\n\n\n## TODO \n- [] Add time stems to output data of arduino\n- [] Data should be written to a CSV directly ot prevent excessive memory usage\n- [] Previous written files shouldn't deleted. Instead, use time stamps for file naming differentiation\n  - [] 
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/Readme.md b/Readme.md
--- a/Readme.md	(revision 64c51f00b72e7e69b26c2cd092c262df788803fd)
+++ b/Readme.md	(date 1642069838251)
@@ -1,10 +1,11 @@
 # Ambient Intelligence PPG Mini-Praktika
 
-Python Code zum Auslesen des Mikrocontrollers
+Mini-Praktikum: Venen Funktionsmessung mittels PPG
 
+Ziel dieses Praktikums ist es herauszufinden ob man mit dem populären MAX30102 Sensor eine Venen PPG Messung durchführen kann.
 
-## TODO 
-- [] Add time stems to output data of arduino
-- [] Data should be written to a CSV directly ot prevent excessive memory usage
-- [] Previous written files shouldn't deleted. Instead, use time stamps for file naming differentiation
-  - [] 
\ No newline at end of file
+# Pull repository including submodules
+
+```
+ git submodule update --init --recursive
+```
